cmd/agent/app/diagnose.go

```diff
@@ -26,6 +28,14 @@ var diagnoseCommand = &cobra.Command{
 }
 
 func doDiagnose(cmd *cobra.Command, args []string) {
+	// Global config setup
+	if confFilePath != "" {
+		if err := common.SetupConfig(confFilePath); err != nil {
+			fmt.Printf("Cannot setup config, exiting: %v\n", err)
```

Comment:
`Println` is probably more adequate here

Reply:
Changed to `fmt.Println`: @mfpierre